pyproject.toml

```diff
@@ -0,0 +1,28 @@
+[build-system]
+requires = ["poetry-core"]
```

Comment:
Any strong reason for using `poetry` as opposed to just `pip`?

Reply:
Any build system world work, it can be replaced with `setuptools`,  `flint` or others. Just this was the system that  I knew how to include optional dependencies and extras. This can be changed.

Reply:
I'd rather use `setuptools`, as all my other projects use that. I can make changes if you like, or I can leave it to you.